conf/services/declare-cash-you-carry-into-or-out-of-great-britain.yml

```diff
@@ -0,0 +1,34 @@
+serviceName: Declare cash you carry into or out of Great Britain
+serviceHeaderName: Declare cash you carry into or out of Great Britain
+serviceDescription: |
+  Use this form to declare cash you carry into or out of Northern Ireland.
```

Comment:
**serviceDescription** shouild be _Use this form to declare cash you carry into or out of **Great Britain**_.